helm/nginx-ingress-controller-app/templates/_helpers.tpl

```diff
@@ -3,85 +3,81 @@
 Expand the name of the chart.
 */}}
 {{- define "ingress-nginx.name" -}}
-{{- .Chart.Name | trimSuffix "-app" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
-{{/*
-TODO remove this legacy helper once switched to "ingress-nginx.name" everywhere.
-*/}}
-{{- define "name" -}}
-{{- include "ingress-nginx.name" . -}}
+{{- default .Chart.Name .Values.nameOverride | trimSuffix "-app" | trunc 63 | trimSuffix "-" -}}
```

Comment:
is `trimSuffix "-app"` from upstream? why is that trimmed?

Reply:
This is from us, it removes `-app` from `nginx-ingress-controller-app` which is the current chart name. So it's basically the same as in the old `name` function. I only renamed it everywhere.

Reply:
But can't we _just_ rename instead our `nginx-ingress-controller-app` to `nginx-ingress-controller`? :thinking: Or is this planned for some breaking changes release later? :smile:

Reply:
Yeah, I feel like this is breaking stuff. Probably the app won't be listed as `nginx-ingress-controller-app` in our catalog anymore and then customers need to modify their `App` CRs.

I'm already planning or at least thinking about how I'm going to release all those breaking changes. Until then and as long as possible I'd just align all the non-breaking things and provide our customers most of the upstream features without the hassle migration.